shpc/settings.yml

```diff
@@ -68,6 +68,9 @@ singularity_shell: /bin/sh
 podman_shell: /bin/sh
 docker_shell: /bin/sh
 
+#Experimental version naming tag so that modulefile is changed to version number . Default value is false. Issue number 26 
```

Comment:
I suggest having this right after `default_version` option, line 39, as they are related